test/molecule-role/molecule/kubernetes/cleanup.yml

```diff
@@ -8,26 +8,32 @@
     - common_vars.yml
   tasks:
     - name: Delete kubernetes test workloads
-      command: kubectl delete -f test_workloads
+      command: kubectl  delete ns {{ agent_current_branch|lower}}
```

Comment:
shouldn't we remove the namespace after all the objects belonging to this names are deleted ?